tests/test_integration_queue.py

```diff
@@ -0,0 +1,200 @@
+import json
```

Comment:
Personally, I think we should have all the integration tests that rely on a running erica in one place, but we can do that when we have the main configured

Reply:
So no todo for this PR?